Automattic/wp-calypso
# Introduce Mailchimp connection

This feature is work in progress.
Works with D19048-code .

Basically this introduces Mailchimp Keyring connection.
**The "Mailchimp" block will only show if your user is an a11n**. - this is blocked in API.

<img width="765" alt="zrzut ekranu 2018-10-04 o 10 34 34" src="https://user-images.githubusercontent.com/3775068/46481309-1cc3a700-c7c1-11e8-8b45-c0387a13835c.png">



## Testing instructions

- apply D19048-code
- Click connect in settings for mailchimp
- Choose a mailchimp list
- Go to incognito window on the site that you set it up on
- Click "Follow" in right bottom corner, subscribe
- see your email in mailchimp list
